pkg/controllers/management/clusterconnected/clusterconnected.go

```diff
@@ -103,6 +103,12 @@ func (c *checker) checkCluster(cluster *v3.Cluster) error {
 		return nil
 	}
 
+	// wait to update the connected condition until it is bootstrapped
+	if v3.ClusterConditionBootstrapped.IsUnknown(cluster) || v3.ClusterConditionBootstrapped.IsFalse(cluster) {
```

Comment:
```suggestion
	if !v3.ClusterConditionBootstrapped.IsTrue(cluster) {
```
This would be sufficient, right?